app/src/main/java/com/example/concordia_campus_guide/fragments/InfoCardFragment.java

```diff
@@ -123,10 +123,10 @@ private void setServicesList() {
     }
 
     private void setOnClickListeners(){
-        this.directionsBt.setOnClickListener(v -> onClickDirections(v));
+        this.directionsBt.setOnClickListener(v -> onClickDirections());
     }
 
-    public void onClickDirections(View v){
+    public void onClickDirections(){
```

Comment:
you sure removing the view doesn't affect anything? There were some other places where it was saying to remove it b/c it wasn't being used and when I did, the app wasn't working properly

Reply:
the reason why the one you touched caused trouble is because they are directly linked to the ui actions. The buttons were not setup in the code but in the UI and were given name of function to look for in the code. Therefore the functions have to conform to the standard of including the view in the argument. This one however does not seem to be linked to a UI element from the xml and should be safe. You can double check